### ISSUE TITLE
InstaPy Ambassador

### ISSUE THREAD

**Issue by timgrossmann** [InstaPy/InstaPy#3574]
#### People who are users of InstaPy but can't contribute with code or any other "business"-skill, can starting being "brand"-ambassadors. Of course only if they are ok with people knowing they use InstaPy. Would be great to, at one point, have a few people with nicely growing accounts that show new users and people interested in InstaPy what you can achieve with the support of it.

Advantages of this feature:
- Better "outlines" of what InstaPy can do
- Showing that using a bot is just normal these days 

Possible ways of achieving this while keeping control:
- Users writing about their experience once they reach good growth (maybe using some, to be created, analytics tooling)

> [Bountysource this Issue](https://www.bountysource.com/issues/67494668-instapy-ambassador)

--- 

###### Feel free to comment with adjustments and improvements

**Comment by cormo1990** [comment]
Yes great Idea @timgrossmann ! I am available for that, count on me

**Comment by converge** [comment]
I'm the first one to join :D

that's funny, I started using InstaPy on this project two days ago, ~40 new followers per day:

https://www.instagram.com/converge_developer/

**Comment by Tkd-Alex** [comment]
Me too! This is my Instagram page :)
https://instagram.com/tkd_alex/

**Comment by cormo1990** [comment]
Hello Fam,
last week has been a year since I started using InstaPy. I want to share my results with you.
using my method (cormo template) I earned 39K followers. 
I have to thank all the people who have and are participating in this wonderful project.
I also learned a lot and I hope that in the next future I can be more helpful for others.

Cormo


@timgrossmann @uluQulu and the others friends 💯

**Comment by Tkd-Alex** [comment]
@cormo1990 Can you share your template? :)

**Comment by timgrossmann** [comment]
@cormo1990 Would be incredible if at some point we could use your profile to show people what can be achieved with InstaPy, congrats on the success! 🙌

**Comment by cormo1990** [comment]
@Tkd-Alex my template is included in the latest release ;) btw it consist in massive follow and defollow by step (first not follower then others).

@timgrossmann yes, i am available for that, below a partial graph of my growth on IG (from March 2018 to January 2019)

![image](https://user-images.githubusercontent.com/36590321/51235638-55524f80-1970-11e9-8de6-53f19455ffa9.png)

**Comment by timgrossmann** [comment]
@cormo1990 This is incredible! 
Thank you very much for the insights.

What is the account about?
Are you interested in collaborating in writing a second part of https://medium.freecodecamp.org/my-open-source-instagram-bot-got-me-2-500-real-followers-for-5-in-server-costs-e40491358340?gi=d9d1c853e10 using your account data?

Is your data very sparse? the points look to be distributed very much in the end?

**Comment by ferninatorcom** [comment]
Hi all,

Wouldn't that be admitting that we are violating Instagram's terms? I might be wrong, but as far as I know Instagram does not approve of automating manual processes. 

@Tkd-Alex congrats on your account! I could see there's been a significant decrease in interaction (likes) in your last posts, do you know what could that be linked to?

@cormo1990 that increase in the follower count is amazing! Could you tell us how many likes are you getting on average in your posts?

**Comment by cormo1990** [comment]
@timgrossmann sorry for the late reply, I was on business trip.
my account is about drone photography, feel free to ask me what you need to know and I will be really happy to be helpful.

@fmorenoper until December I reach from 5K to 9K likes for posts but when instagram changed the algorithm I lost a lot of audience, I don't know why exactly, now I reach from 2K to 5K likes for post
